v0/pipeline.json

```diff
@@ -44314,7 +44325,18 @@
               "description" : {
                 "desc" : "This is the description for SyncStepInfo"
               }
-            }
+            },
+            "anyOf" : [ {
+              "oneOf" : [ {
+                "required" : [ "applicationRegex" ]
+              }, {
+                "required" : [ "applicationsList" ]
+              } ]
+            }, {
+              "not" : {
+                "required" : [ "applicationRegex", "applicationsList" ]
```

Comment:
what does the step do if both are absent?

Reply:
If there is a gitops app from previous step(UpdateGitopsApp, FetchLinkedApps) which is present in sweepingoutput that app is by default appended to list of apps that will be synced